app/models/exercise/taught_concept.rb

```diff
@@ -0,0 +1,6 @@
+class Exercise::TaughtConcept < ApplicationRecord
+  belongs_to :exercise
+  belongs_to :concept, # rubocop:disable Rails/InverseOf
```

Comment:
We don't need this inverse_of as we don't have an association on the other end at the moment.